texar/torch/modules/tokenizers/pretrained_gpt2_tokenizer_utils.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utils of pre-trained GPT2 tokenizer.
+"""
+
+from functools import lru_cache
+
+__all__ = [
+    "bytes_to_unicode",
+    "get_pairs",
+]
+
+
+@lru_cache()
+def bytes_to_unicode():
```

Comment:
This should be under `tx.utils.types`. 

I believe Py3 does not have the concept of `unicode`. A counterpart in Texar TF is the `compat_as_text` function: https://github.com/asyml/texar/blob/master/texar/tf/utils/dtypes.py#L118

Reply:
I feel that this is not a general-purpose function. The dictionary mapping generated from this function does not include some special characters such as whitespace and control characters which are specifically related with the gpt2 `bpe` mechanism.